tagilmo/VereyaPython/load_from_xml.py

```diff
@@ -0,0 +1,223 @@
+import xml.etree.ElementTree as ET
+import tagilmo.utils.mission_builder as mb
+from .xml_util import remove_namespaces
+
+def _createAbout(aboutRoot = None):
```

Comment:
I think we can move these helper functions into corresponding classes, e.g. createAbout into About.from_xml, _createAgentSection into AgentSection.from_xml etc..

@CICS-Oleg what do you think?